src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/courtsentencing/CourtChargeMappingDto.kt

```diff
@@ -6,13 +6,13 @@ import jakarta.validation.constraints.Size
 import java.time.LocalDateTime
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@Schema(description = "Court Charge mapping")
+@Schema(description = "Offender Charge mapping")
 data class CourtChargeMappingDto(
```

Comment:
Offender charge is a better description than court as it is also used as a sentence mapping. Will rename in the future as will inflate the PR a bit too much